site/content/contribute/mobile/e2e/file-structure.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Folder and File Structure"
+date: 2020-09-02T09:00:00-00:00
+weight: 2
+subsection: Mobile End-to-End Testing
+---
+
+The folder structure is mostly based on the [Detox scaffold](https://github.com/wix/Detox/blob/master/docs/Guide.Jest.md) which was created on initial run.  Folders and files are:
+```
+|-- detox
+  |-- e2e
+    |-- support
+    |-- test
+    |-- config.json
+    |-- environment.js
+    |-- init.js
+  |-- .babelrc
+  |-- .detoxrc.json
+  |-- package-lock.json
+  |-- package.json
+```
+
+1. `/detox/e2e/support` or [Support Files](https://github.com/wix/Detox/blob/master/docs/Guide.Jest.md#2-set-up-test-code-scaffolds-building_construction)
```

Comment:
Is this a reference to a specific requirement or just a link for more information?

Reply:
@justinegeffen Link for more information